public/index.html

```diff
@@ -19,7 +19,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Github Battle</title>
```

Comment:
You'd renamed your app in the index.html in the src directory, but that renaming should happen here.